### DISCUSSION TITLE
Feedback for new issue viewer in projects

### DISCUSSION THREAD

**Discussion by azenMatt** [community/community#62400] [Projects and Issues]
### Select Topic Area

Show & Tell

### Feature Area

Issues

### Body

We just released (or are in the progress of releasing if you are still seeing the old one) a new issue viewer for when you open an issue inside of a project. 

This issue viewer now includes: 
- Timeline events 
- The project picker (so you can edit projects in addition to the current project)
- Faster performance 

Most importantly this is a first step to having a single implementation of the issues page we can use across GitHub. In the future this page will be coming to the repository so we would love your early feedback! 

What are your thoughts on this new issue viewer? Is anything missing vs the old one? What could be better? What do you like?

We will continue to release updates as we work on parity against the main repository issue page (eg transfer an issue) on a regular cadence. 

The below is an example from our [public roadmap](https://github.com/orgs/github/projects/4247/views/7?pane=issue&itemId=30227707) of how the new issue view looks:

![image](https://github.com/community/community/assets/7584089/085e20f3-5293-4f29-ae5d-e5eec1aaa6d4)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by daveyholler** [comment]
I'd like to see links be clickable in the project details section of the sidebar:
![CleanShot 2023-08-08 at 13 13 41@2x](https://github.com/community/community/assets/739960/d9721176-cf5f-4d92-905f-a09b91898718)

**Reply by azenMatt** [reply]
Good spot - we will get that updated

**Reply by Darlfas1** [reply]
Thanks so much for the hard and good works. I can't wait to see us done and dusted in readiness for Mainnet.

**Comment by janakerman-form3** [comment]
Previously when I used to copy + paste the title of an issue I was viewing, I'd get a nicely formatted link along the lines of [My Issue Title#0000](url). This made it really easy for me to paste the issue title with a link. That no longer seems to work in the newer view. Thanks.

**Comment by joerj123** [comment]
It all just feels so sluggish. Adding labels or tickets to task lists takes forever. Please make it faster!

**Reply by azenMatt** [reply]
Hi @joerj123 - we just pushed an update to improve the performance for adding labels and milestones. Does this feel better to you now?

**Reply by bprice6** [reply]
Updates still lag for me >5 seconds to the point where it makes me wonder if I added or not.

**Comment by miyataka** [comment]
I want the behavior of editing an issue to be the same as the behavior of editing an issue from a project.
Specifically, I want tab input in the text area when tab is pressed. Now the latter moves focus away from the textarea.

Thanks.

**Reply by azenMatt** [reply]
👀 this is how issues have always worked. If you are in a comment-box, tab will take you outside the comment box. 

Or am I miss-understanding the component you are referring to here?

**Reply by miyataka** [reply]
@azenMatt 
Thank you for your reply!!!
Something like this.
- [video0](https://github.com/community/community/assets/15692588/b4df72e7-00cf-48de-9183-39b089877535)


- [video1](https://github.com/community/community/assets/15692588/cfbab39d-fd97-43d0-9b21-092bce2601ef)

**Comment by zaguiini** [comment]
I do not always want to open the issue as a modal. Previously, the issue title had a link. Now, only its number has it. It was easier before, and I can't see a good reason not to bring this wider clickable area back.

**Reply by Alfagun74** [reply]
also related: https://github.com/orgs/community/discussions/62400#discussioncomment-8238950

**Comment by rainbow1016** [comment]
very good question!

**Comment by juanmaguitar** [comment]
Hi @azenMatt! 
We're using a project to link issues and PR's from different repos and organizations with custom fields. We seem to have lost the capacity to access these custom fields for the issues that belong to external repos.

**Reply by juanmaguitar** [reply]
> **Warning**
> **TL;DR** - It seems there's been a change in GitHub projects that doesn't allow to have issues in a project where  the issue belongs to a different organization or already belongs to another project

To provide more context, I will explain this issue with a few screenshots:

As a team, we have our own project to organize our work. This project belongs to the organization "Automattic". In our custom project we add issues from different repos and some of them belong to different organizations (like "WordPress")

In the following snapshot, you can see an issue (from [WordPress](https://github.com/WordPress) / [developer-blog-content](https://github.com/WordPress/developer-blog-content)) added to our project dashboard (at [Automattic](https://github.com/Automattic) / [Projects](https://github.com/orgs/Automattic/projects) / [Bifrost Sprints](https://github.com/orgs/Automattic/projects/493)). 

In the snapshot, it can be seen how this issue belonging to another org/repo contains the value `sprint:"🅱️ Sprint Q3-4"` as one of the custom fields we defined exclusively for our project.
<img width="516" alt="Screenshot 2023-08-25 at 10 14 44" src="https://github.com/community/community/assets/422576/fbbc7c79-536d-4bbb-8929-b1fb5909176d">

If we click on the issue we get the following card of the issue that doesn't contain either our project or any of the custom fields we defined for that issue for our project. If we try to change the project, our [Bifrost Sprints](https://github.com/orgs/Automattic/projects/493)) doesn't appear in the list of available projects for this issue. 

In summary, **we cannot access, view or edit, the custom fields we defined for our project in issues from other orgs/repos which was something we could do last week,** and it was one of the main reasons we chose GitHub projects to organize our work (as we deal with issues from different repos)

<img width="1264" alt="Screenshot 2023-08-25 at 10 15 02" src="https://github.com/community/community/assets/422576/b4fa9794-376a-48b3-b8fc-9545384b7658">

**Reply by japf** [reply]
👋 @juanmaguitar thanks for the report, we're currently taking a look 👀

**Reply by japf** [reply]
@juanmaguitar our team just shipped a change to address this feedback. To be more specific: when you view an issue in `orgA` from a project that is owned by `orgB` we'll now always render all projects from `orgA` _and_ projects from `orgB`. If you view the issue directly, we'll only render project from `orgA`.

Please let me know if that is fixing your scenario 🤞🤞🤞

**Reply by juanmaguitar** [reply]
Hi @japf, 
Yes, the issue is solved now 👍
Thanks a lot for the fix!

**Reply by Darlfas1** [reply]
That was a great report. Thanks so much.

**Comment by Imran92** [comment]
When creating an issue, if I try to add a label, I can select a label from the dropdown but the selected label doesn't appear under **Labels** in the Modal

![image](https://github.com/community/community/assets/6820724/6ecee98d-35e1-4ebb-92dc-3ee4595e2de7)

**Reply by japf** [reply]
👋 Hey from the Issues team, I'm not able to repro this scenario. Could you share the steps you're following? You mention "when creating an issue" but it looks like your screenshot is about _editing_ an issue?

**Comment by kraftbj** [comment]
Thanks for adding clickable links in the sidebar for custom fields... but, for the life of me, I couldn't find a target to click on to edit it. We imported everything from Asana and removing some old Asana links. I ended up inspecting the DOM, deleting the link, and expanding the `<button>`'s height in order to trigger the button for editing.

**Reply by azenMatt** [reply]
Thanks - yes this isn't working as well as we want it to and has also caused some accessibility concerns. We are going to need to revert it but will find a better implementation

**Comment by mctaylorpants** [comment]
I just noticed a small UI issue which I think might be a regression, since I believe this was working the other day:

When editing the "description" field of an issue in the issue viewer, the textarea is missing the handle on the bottom-right that allows you to expand the textarea. Here's what it looks like in the issue viewer:
<img width="207" alt="Screenshot 2023-08-29 at 15 35 36" src="https://github.com/community/community/assets/7259082/69783b0d-b7b0-43b2-a461-1e20bed5c4ff">

... and here's what it looks like in the normal full-screen page:

<img width="283" alt="Screenshot 2023-08-29 at 15 39 18" src="https://github.com/community/community/assets/7259082/61d4ac95-4cdb-4775-8ad6-779504f155cc">



Super minor thing and probably browser-specific, but it does seem like it might be cut off on the issue viewer.

**Reply by japf** [reply]
👋 Thanks for reaching out, yes this is a regression that happened yesterday. We're about to deploy a fix 🤞

**Reply by japf** [reply]
Resizing the comment box is back 🎉

**Comment by jasell** [comment]
The new layout requires way too much scrolling!

- In the new layout the key information of the issue is to far down.
I want to see the Assignee, Project section in the top.
Either below the title or after the fist post.

- To improve layout you can reduce fontsize for the issue title as well
Currently the title consumes 2 to 5 rows in large font, which is a big part of the screen

**Reply by Antzed** [reply]
+1, the key information needs to be on the top

**Reply by wafer-bw** [reply]
+1, things are too bloated. Elements need to be more compact.

**Comment by vforvova** [comment]
Previously select lists for custom project fields had gray text with descriptions to the side (after the actual item name), and now they are missing. Would be nice to have them back - they help quite a bit.
![image](https://github.com/community/community/assets/1913904/d9beaba9-dd11-483e-8b33-e42e0595275b)

**Reply by japf** [reply]
👋 Thanks for the feedback. We just shipped a change to add those descriptions 🚀

**Comment by SydAirey** [comment]
Why did you guys take away the copy link button?!  Please put it back!

**Reply by MarkBennett** [reply]
Definitely agree with this. It's super helpful to have a quick way to copy a link for sharing with teammates without needing to open a seperate tab and copy the issue url from there.

**Reply by azenMatt** [reply]
This one? 

![image](https://github.com/community/community/assets/7584089/1b68f1b8-6516-46fa-981c-d1f1b0fd3bee)

**Reply by SydAirey** [reply]
I noticed that too!  Just have to get used to that change but glad it didnt fully go away. Thanks!

**Reply by MarkBennett** [reply]
Ha, I didn't even notice that. Thanks for the tip @azenMatt! 👍🏼

**Reply by francescolaffi** [reply]
Was going to ask the same, glad it's just moved 👍  not sure if the icon is self-explanatory for everyone though

**Reply by mishmanners** [reply]
Oh neato. I'll be using that.

**Reply by jayalberts** [reply]
Had mild panic attack thinking copy button was gone 😅

**Reply by treeder** [reply]
I had the same panic attack until I found that copy button. 😆

**Reply by EdwardSalkeld** [reply]
Same. I use this _a lot_. Totally missed that.

**Comment by bobcats** [comment]
One of the reasons I loved having "open in a new tab" easily available was because it's only possible to quickly create "sub-issues" from a checklist on an issue on the full issue show page. If we could bring that experience to the issue panel in a project, that would be sweet!

**Reply by japf** [reply]
👋 Thanks for sharing this feedback, we're aware and we just started working on that. I don't have a timeline to share yet but we agree this is an important feature we need to have in the new issue viewer 🚀

**Comment by mishmanners** [comment]
This looks really cool! Can't wait to try it out.

**Comment by Antzed** [comment]
Can you guys add the option to hide the timeline item in the issue viewer for those of us that automate issue labeling using github actions? Sometime it can be hard to find the specific comments when the timeline items took over most of the screen space?

**Reply by garrettn** [reply]
+1. I've been working off a checklist that's a comment on one of our project issues, but now it's buried in a long list of timeline events (mainly from commits that reference this issue). I used to prefer this view because it would hide the events. If I wanted to see all the events I could always open the issue in a new tab. If there could be a way to toggle that or something, that would be wonderful.

**Comment by azertyalex** [comment]
I like the new extra functionality a lot!
When I try to create a branch for my issue the layout is a bit cut-off
![image](https://github.com/community/community/assets/9082988/5a6743d3-1f74-4b0a-a688-32624542312a)

**Comment by EToddS** [comment]
I think the new layout is an improvement overall, my only complaint is that the "Status" field used to be shown in the color scheme it was assigned to (our "In Progress" status is yellow-orange, so the text "In Progress" on the issue's side bar would be yellow-orange). It no longer does this and is just plain white text. This makes it less noticeable in my opinion.

**Reply by japf** [reply]
👋 Thanks for the feedback, we shipped a change yesterday with that 💟

**Reply by EToddS** [reply]
Thank you!

**Comment by Christian-Toney** [comment]
It would be nice to find out the full text of a truncated field is by hovering over it.

![image](https://github.com/community/community/assets/37517470/11363538-848f-45df-98d6-de097e768d64)

**Comment by sctice-ifixit** [comment]
It would be really nice to have ctrl+shift+p toggle between edit and preview when editing the description, as it does on the standalone issue view. This shortcut has always been missing from the project issue view, as far as I'm aware, but I'd love to see it added.

Separately, it would be really nice to be able to reorder todo items (list items with a checkbox) in the issue description, another feature found on the standalone page.

**Comment by smagret** [comment]
When editing one of the custom fields from the project, you have to manually press the confirm button, pressing the enter key does not work.
Previously, you could click anywhere else on the modal and the value would auto confirm.
I've seen that if you close the modal it also saves the value, so maybe the value still auto saves itself but the UI does not give feedback as if the value is confirmed and saved.

**Comment by smagret** [comment]
If you open a referenced issue from the modal of an opened issue, the back button stops working.

![image](https://github.com/community/community/assets/82170493/9e94a531-ba8b-4494-81c0-d640951d477c)

It only happens when you open the link in the same page where you have the project opened.
The URL and title of the tab changes to the previous page (the project) but the page itself does not change and stays in the opened (referenced) issue page.

**Comment by Flucadetena** [comment]
Hi everything looks great, just one thing I noticed is that when you change the "Status" inside the issue it takes a lot of time to update the table. But viceversa doesn't happen.

When you update here:
![Captura de pantalla 2024-01-18 a las 12 35 42](https://github.com/community/community/assets/38214790/fdcd984f-1641-4f3a-8b80-707820d90480)

This section takes ages to refresh:
![Captura de pantalla 2024-01-18 a las 12 35 53](https://github.com/community/community/assets/38214790/e262967d-dc5f-4076-945a-65eacf486341)


Hope it helps, thanks for the great work!

**Reply by emripka** [reply]
bump on this

**Comment by treeder** [comment]
I just got the new interface, and previously it had a big "tracked by" chip at the top of an issue so you could easily see it's part of a bigger issue and navigate to it. Now that has moved way down the right hand side which makes it barely even noticeable. 

Used to be up here: 

![image](https://github.com/community/community/assets/75826/6951d064-e43a-46e5-8ba3-80ea12dda7c9)

Now it's way over here: 

![image](https://github.com/community/community/assets/75826/ab9c1516-028a-4419-aa68-c18bcf5beaf6)

Would be nice to have the link back to the main issue at the top again.

**Reply by eeberhard** [reply]
I agree. I came to this page to give feedback on this point. I don't like that the "Tracked By" chip is gone. In many cases, I have one "epic" issue with a task list to group other issues as sub-tasks. The previous view with the "Tracked By" chip was better in several ways:

- Make it easy to see immediately the tracking issue at the top 
- Have a direct link to the tracking issue
- "Tracked By" made the direction of the dependency very clear, while the new "Relations" section is not obvious in how the issues are related

**Reply by knudsvik** [reply]
Agreed! I don't even have the "Relations" section so impossible to see the relations. Would like it to be on top like a chip, same way as opening it in a new tab.

**Reply by knudsvik** [reply]
This seems to have been fixed now!

**Comment by archon810** [comment]
Thank you for finally adding timeline events and opening the project dropdown by default. I've been waiting for it since the new projects were released.

**Comment by archon810** [comment]
Found a bug with the new UI. If I open ticket #123456 which references ticket #654321:
1. Ctrl-click on #654321 while viewing #123456 in a project modal doesn't open #654321 in a new tab and still loads it in the modal, replacing the currently open ticket. Ctrl-click should always open in a new tab and leave the currently open ticket alone.
2. Clicking Back in the browser switches the url back to `...&pane=issue&itemId=123456` and even the page <title> switches back to the correct one for #123456, but it doesn't actually load it in the UI. The UI continues to show #654321.

**Reply by archon810** [reply]
@japf Have you been able to reproduce these?

**Reply by japf** [reply]
Yes I was able to repro and I created an issue internally. Not sure about the timeline for a fix yet.

**Comment by ghost** [comment]
The label text colour isn't adjusting depending on the background (using #532E33 here):
![image](https://github.com/community/community/assets/114012401/ffb92643-6346-4d89-a0a7-030d14941f62)

And the same on the timeline:
![image](https://github.com/community/community/assets/114012401/67aa90bc-33cf-4467-b7a0-df5e44c6ab5d)

On the board view:
![image](https://github.com/community/community/assets/114012401/d3507c88-32e7-4b84-b0df-617b91bd450f)

**Reply by boazreicher** [reply]
Hi @ben-propflo, we're not able to replicate this.  When testing with the same color label (`#532E33`) and light theme, the text is rendered white.  Could you share the specific theme and browser you're using?

**Reply by ghost** [reply]
Hi @boazreicher 

I'm using the day theme, Light default with 'Sync with system'. Firefox 121.0.1 on Fedora.

Here's where that text colour is being applied on that specific tag:
![image](https://github.com/community/community/assets/114012401/8dc9f582-43c4-435a-9c01-d0aa7c00c4f9)

Let me know if you want me to grab anything else to assist you.

**Reply by japf** [reply]
👋 I can repro in Firefox, thanks for sharing the details.

**Reply by btmills** [reply]
Using the light theme (synced with system), I'm able to repro dark label text on dark backgrounds in Firefox 121.0.1 and 122.0 on macOS 14.2.1. The label text is the expected light color on dark backgrounds in Chrome and Safari.

**Comment by miked1ck** [comment]
I wish the new project fields were not so far down the page now. When inside a project it's so important for quick access to them when in context of a project. Now, it feels like hunt and pecking for it but I may get used to it.

**Comment by connorjak** [comment]
I don't like how "open in new tab" is now in a context menu; if it could be linked from the issue title text that would be better

**Reply by connorjak** [reply]
Though, now the preview page is pretty much at parity with the new-tab page now, so there isn't much reason to open in new tab.

**Comment by kjy5** [comment]
I got the new UI! I love how I can finally open development branches for issues directly in the projects interface. However, there appears to be a bug where branches created from the projects interface don't automatically link to closing pull requests on the branch. If I create the branch from the normal issues tab, it works fine, but when created from the UI in projects it doesn't. With that fix then the whole interface would be perfect!

**Comment by TouchstoneTheDev** [comment]
Content from projects could be searched in search. i haven't seen it yet !?

**Reply by TouchstoneTheDev** [reply]
I am using this type of dork "site: https://github.com/ inurl:https://github.com/orgs/*/projects/"

**Reply by TouchstoneTheDev** [reply]
@azenMatt Can't project scope be added in github search or seprate.

**Comment by kentatakase** [comment]
It used to be a one-click EDIT.Now it requires two clicks. A little stressful.
![image](https://github.com/community/community/assets/77864325/e6ba5896-893c-4dfd-9edb-b3468b1dbb2c)

**Reply by Alfagun74** [reply]
yeah i also think editing an issue is one of the main use cases and the pencil could be just put next to this three dots icon. Also would be nice because the new "Edits" history button is also there.

**Reply by pabloportugues** [reply]
Quick access actions should have the fewer steps possible, and as mentioned here editing a description of an issue is one of those actions. Please make it one-click edit!

**Comment by loraxster** [comment]
The new alpha now hides other assignees so I **have to** start typing their name. Since our group is small, it is more efficient to see the list so I can simply click on the checkbox. Please fix this back to the old way.

**Reply by florian-lerch** [reply]
I second this; especially when you want to assign multiple people it's tedious to find all their names.

**Comment by jasell** [comment]
The alpha version do not let you add new Fields to Project features. Befors you could search/filter, and if you didnt’t got a match you had an option to add new field. Now you need to open settings and navigate to the property, add the field, then back to the first page, reload the page to load the new option, then you can select it. Major drawback!!1!!

**Reply by japf** [reply]
👋 Thanks for the feedback, I assume you're referring to the ability of creating a new value in an existing field, is that correct?

<img width="395" alt="creating option from a non existing value" src="https://github.com/community/community/assets/2237018/1b621769-ad9c-4d74-818f-fa849cd4ad7f">

**Comment by iamsid** [comment]
When creating a new issue in projects, the description input requires two clicks to edit it.
The previous UI just had an editable input ready to setup the description.

Issues UX has always been like this. But now that UIs are merging, the description UX could be improved.
For a new issues (from projects or issues), if there is no description, the description could always be editable right away.
But once it has some text saved, two clicks to edit would be ok.

**Comment by phifuh** [comment]
I like the design & this is a nice feature, but honestly this view should be optional or there should an alternativ "simple view".
The clean minimalistic view, is for some projects and for some groups of people better. 

Once you moved the issue 10x and/or changed labels, the comment history just gets bloated.
Following up on, what everyone was saying gets harder

**Reply by loraxster** [reply]
Is there a way to hide this history-of-everything-on-this-issue thing? we never want to see it (no use for us).

**Reply by Antzed** [reply]
+1

**Comment by nikitaeverywhere** [comment]
A very wanted but missing feature is "duplicate", which would create the same issue with same properties but maybe another title.

It would be very convenient for creating a lot of tasks in one go with the same template, for instance, during planning or grooming.

![image](https://github.com/community/community/assets/4989256/45992079-e093-4951-9b85-6bc85603448b)

[Related feature request](https://github.com/orgs/community/discussions/8485)

**Reply by MarkBennett** [reply]
Perhaps it might make sense to allow quickly creating a new issue template from an existing issue?

**Comment by raethkcj** [comment]
When pushing multiple commits that reference an Issue that's tracked in a Project, the first commit is shown duplicated by the number of commits in the new Projects Issue Viewer:
![image](https://github.com/community/community/assets/7716908/6ac180fd-74f2-4374-93e9-eac217644188)
The regular Issue page for the same Issue shows the expected individual commits:
![image](https://github.com/community/community/assets/7716908/21e1bb0b-66e2-4df8-8459-b71460777716)

**Reply by martin-weidhofer** [reply]
Experiencing the same

<img width="651" alt="image" src="https://github.com/community/community/assets/105639806/fa0d83c2-7f2c-4e86-8db9-351af0171405">

**Reply by japf** [reply]
👋 Thanks for the report, we're taking a look and we should have an update soon 🤞

**Reply by japf** [reply]
This has been fixed 🎉

**Comment by bosantosS** [comment]
I am currently managing more than one project but I feel that the content of the charts is very limited.
Additionally, I would like to be able to download the backlog in the form of excel or csv document tables for further data processing.

**Comment by Alfagun74** [comment]
When im scrolled up i can only click the issue number but not the issue title:
![image](https://github.com/community/community/assets/22225051/ddcddcfa-e1e6-415d-b0bf-8231dd7ee9e5)

When i scroll down i can only click on the title but not on the issue number:
![image](https://github.com/community/community/assets/22225051/287553a4-7035-4620-b962-4e7a17b12f6a)

Personally i would like to be able to click both to open the full issue.

**Comment by Antzed** [comment]
The convert to issue from tasklist function does not work in the project issue view. Currently if I want to use that function I will have to open the issue in a new tab and do it from there. It would be great if issue convertion is added into the project issue view.

**Comment by machty** [comment]
I wasn't able to find a simple link an Issue when viewing it within a project until I came to this feedback thread and realized you have to click the `#123` issue number in the header. I feel there should at least be some other link, ideally next to the link to the repo issues that's near the header.

**Comment by rr-aiden** [comment]
It would be nice if when clicking on a custom number field in the projects UI, that clicking off without entering something should not result in a default value of 0. The current workaround to resolve this is being forced to enter a value, or going to the issue and removing it directly from that screen - seems a bit clunky.

**Reply by japf** [reply]
👋 @rr-aiden would you mind capturing a small video for that? I'm not able to reproduce 🤔

**Comment by ghost** [comment]
I would like the various Copy... links back please
although I have just realised they are hidden in the ellipsis at the top of the page...it's too hidden IMHO

**Comment by gigpiged** [comment]
Not sure if this is in scope of this - but can we have the ability to en-mass move issues across iterations?  Currently we need to move all our issues one by one for any incomplete issues in that sprint/iteration.

**Reply by japf** [reply]
👋 Hey @gigpiged one way to do that is to set one iteration and then use drag and drop to set that value to more item. Here is a small demo:


https://github.com/community/community/assets/2237018/c06cc386-31b2-41ce-a23a-5fb71a806e51

Another way is to copy the iteration cell you want (<kbd>Cmd+C</kbd>) and paste that on other cells (<kbd>Cmd+V</kbd>)

**Reply by gigpiged** [reply]
Oh okay! Completely missed that - is this in this latest release or have I just not noticed it for the past year!?

**Reply by japf** [reply]
I think this shipped at some point last year 🚀

**Comment by sramekmichal** [comment]
Hi, I am using my repo as a laboratory notebook during my PhD.

It would be nice, if GitHub would recognize `.md` file provide formatting toolbar automatically.

Toolbar is visible in Comments or in Issues perfectly:
<img width="500" alt="Snímek obrazovky 2024-02-02 v 22 36 38" src="https://github.com/community/community/assets/115155195/46a558cc-560e-40e1-8f80-aa0dca402bb2">


But when writing notes or documentation, I need to format things manually - which of course works, but it just slow:
<img width="500" alt="Snímek obrazovky 2024-02-02 v 22 35 45" src="https://github.com/community/community/assets/115155195/1e9b5f29-69f1-4345-82e1-24ee1681a625">

**Comment by sarahsporck** [comment]
It seems the issue labels aren't accessible in the sidebar issue viewer. Black text on black background is barely readable.
<img width="852" alt="Bildschirmfoto 2024-02-06 um 16 39 30" src="https://github.com/community/community/assets/28013368/914e4049-944f-47f9-95c0-154d604223bd">

**Reply by japf** [reply]
Thanks for the feedback @sarahsporck we're aware and we have a fix almost ready for deploy 🚀

**Reply by japf** [reply]
The fix was shipped in production yesterday 🎉

**Comment by ollejernstrom** [comment]
You can convert an issue with unsaved changes. Both the confirm/discard modal and the repository selector is visible. Upon converting you lose the progress. Perhaps you should not be able to convert until having answered.
![image](https://github.com/community/community/assets/113042755/b031eaf9-19b3-4d3a-8256-b329341df4d7)

**Reply by japf** [reply]
Thanks for the feedback @ollejernstrom I just created an issue internally for this bug.

**Comment by ollejernstrom** [comment]
I feel like everytime i close the drawer i would like a "save & exit" feature. Currently both the "X" and "Keep editing" does the same thing. Just so that if you want to save and click outside you dont have to, close, save, click outside. 

Not sure if this is only me but if the "Keep editing was switched to "Close and save" even though i think the labels should be the other way since the import thing is if it should save or discard. In short, i'd like it if the labels were "Save and Close" and "Discard and Close", and the X for keep editing.

![image](https://github.com/community/community/assets/113042755/fd96b4da-cd63-4b50-b7ce-b0531578a94f)

**Comment by Flucadetena** [comment]
Hi something that has been very tedious for our team since the beginning is that the project does not appear on their "Your projects" section. Even if they are a member of the project.

We tried moving the project to an organization but all the issues and assignments to the members would be lost. So for now their are stuck having to enter my page and searching for the project.

It would be nice if the projects you are a part of just appear on "Your projects" section, even if you are not the owner.

**Reply by azenMatt** [reply]
Does the global projects page accessible via the left hand nav menu help here? 

![image](https://github.com/community/community/assets/7584089/02552e68-28a2-4867-a91e-2ce37037f929)


IMO this is a bit of a  confusing navigation pattern, but in general all the links via the left menu are global and the right menu are personal. 

![image](https://github.com/community/community/assets/7584089/bd6bab2b-5a2e-434f-a51f-d873e99bd7c7)

**Reply by Flucadetena** [reply]
Ohhh! Great thanks! That is very confusing indeed. Hope they change it.

**Comment by Flucadetena** [comment]
It would be very nice to have the ability to link tasks with each other in the roadmap view. Making them dependent of each other. So if one is moved / delayed, the rest of the task move forward that same amount.

It would make roadmap updates sooo much easier.

Thanks for the great work, our team is loving the tool.

**Comment by kazwata** [comment]
Tasklist block removal dialogue is too wide...

![Slack_-_engineering-general_-_kii_🏠](https://github.com/community/community/assets/1888633/2317746e-1792-4a26-824e-7ea497e12e8b)

**Reply by japf** [reply]
👋 Thanks @kazwata for the feedback. We're exploring a newer iteration of tasklist blocks right now so I don't think we'll revisit this in the short term. Please continue to share feedback though, it's super useful to us on the Issues team 🙇

**Comment by SydAirey** [comment]
Not sure if this was mentioned already because there is a lot of feedback here but I notice that my custom sprint field dates do not match with what pre-populates from the quick view.  I have attached a few examples to show below: 
<img width="480" alt="Screenshot 2024-02-06 at 1 09 16 PM" src="https://github.com/community/community/assets/129767528/a72102bb-fe25-4ee0-b227-7f27e62d6c58">
The first screen shot are the custom dates I chose in the workspace settings but when I am in the quick view adding issues to the project and their sprints, it shows the pre-populated sprint fields.  see below: 
<img width="321" alt="Screenshot 2024-02-06 at 1 08 54 PM" src="https://github.com/community/community/assets/129767528/715ca261-2f42-40f2-8414-a0abde256a8f">

Its only a day off and not a huge deal because once I go into the issue itself, it reflects the custom date ranges but just wanted to make sure you guys knew about it.

**Reply by japf** [reply]
👋 @SydAirey that's a good catch, I just created an issue internally for us to take a look.

**Comment by macMikey** [comment]
am i dumb? did i forget how to CRUD labels?
i'm editing via the card view. i click on the gear to edit labels and...i see labels that i can assign to the card, but i don't see how to CRUD labels. i thought that was where we went to do that? if it isn't, what is the gear for?
i thought clicking on the label "labels" was for assigning labels, and the gear was for CRUD.
i found the label mods in repo issues, but it would be much nicer if we could modify labels in the project, again.

**Reply by SydAirey** [reply]
You have to open it in a new tab to see the edit labels button

**Comment by snatvb** [comment]
On real project not enough templates for tasks and duplicates - would be great to have it

**Comment by markcarroll** [comment]
Looking great. Still not able to enter emoji such as :bug: etc into the title bar with autocomplete replacing with the 🐛 character, etc.  We still have to open up the full page to do that.

**Reply by japf** [reply]
Thanks for the feedback @markcarroll, I just created an issue for us internally.

**Comment by jay-babu** [comment]
![image](https://github.com/community/community/assets/36803168/4dc06080-aa55-4de7-b6e6-cbb66daa013f)

Previously, creating a branch and then creating a pull request for the branch would update the issue to link the pull request instead of the branch itself. this no longer works correctly. i have to manually link the pull request

**Reply by azenMatt** [reply]
Thanks for sharing - we will get this fixed

**Comment by human-bee** [comment]
When. I convert a ticket in Projects to an issue, I'm no longer allowed to assign the ticket to anyone.

**Comment by ken-tan3** [comment]
Do not embed a feature "press enter once&save the content" because people who need to convert local lang from alphabet cannot type in correct. It occurred in the title in split board with alpha version.

**Comment by loraxster** [comment]
Something broke recently. When I create a new item, then click on it, edit, save changes  - then try to Convert to issue - this pop-up appears and won't go away. Pop-up should not be appearing (I saved my edited changes).
With the pop-up showing, now I need to go ahead and select the repository. then click the X on the pop-up to make it go away.

![image](https://github.com/community/community/assets/20292342/42abcc5d-b499-406a-96fe-b8365f75355c)

**Comment by alexz707** [comment]
Is there a way to see the history (like in a normal issue) for fields which are only present in the project view?
E.g. we have a prio field which is only in the project view and not on the issue itself.
At first the ticket does not have a prio when it was created. If someone now adds the prio "high" I do not see when & who changed this field.

**Comment by Aboodkaaboodkaaaa** [comment]
؟

احصل على Outlook لـ iOS<https://aka.ms/o0ukef>
________________________________
من: Al3x Zamponi ***@***.***>
‏‏تم الإرسال: Monday, March 4, 2024 12:25:45 PM
إلى: community/community ***@***.***>
نسخة: Subscribed ***@***.***>
‏‏الموضوع: Re: [community/community] Feedback for new issue viewer in projects (Discussion #62400)


Is there a way to see the history (like in a normal issue) for fields which are only present in the project view?
E.g. we have a prio field which is only in the project view and not on the issue itself.
At first the ticket does not have a prio when it was created. If someone now adds the prio "high" I do not see when & who changed this field.

**Comment by trusktr** [comment]
I don't see a way to add an issue link (f.e. so that one issue is blocked by another issue). I see the ability to link pull requests, which is similar. This would link issues to make issues dependent on other issues.

For now, I state this in the issue description.

**Comment by akleinvm** [comment]
Can we make the audit trail collapsible? It can be distracting—especially when there’s no other information on the issue.
